src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -351,29 +351,19 @@ class ExternalDatabaseManagementService(
     }
 
     /**
-     * Sets privileges for a user on an organization's table or column
+     * Sets privileges for a user on an organization's column
      */
-    fun executePrivilegesUpdate(action: Action, acls: List<Acl>) {
-        //split acls between table and column objects
-        val aclsByType = acls.partition { it.aclKey.size == 1 }
-
-        val tableAclsByOrg = aclsByType.first.groupBy {
-            organizationExternalDatabaseTables.getValue(it.aclKey[0]).organizationId
-        }
-        val columnAclsByOrg = aclsByType.second.groupBy {
+    fun executePrivilegesUpdate(action: Action, columnAcls: List<Acl>) {
+        val columnAclsByOrg = columnAcls.groupBy {
             organizationExternalDatabaseColumns.getValue(it.aclKey[1]).organizationId
```

Comment:
Might be worth to batch this call with a predicate or entry processor and keep it in a separate map with `aclKey[1]` as keys and ` OrganizationExternalDatabaseColumn` as value.
It is also used later in `getTableAndColumnNames`, so that map could just be passed there and we could avoid having multiple calls to the hazelcast map.